ndmg/stats/qa_reg.py

```diff
@@ -154,14 +170,13 @@ def plot_brain(brain, minthr=2, maxthr=95, edge=False):
 def plot_overlays(atlas, b0, cmaps=None, minthr=2, maxthr=95, edge=False):
     plt.rcParams.update({"axes.labelsize": "x-large", "axes.titlesize": "x-large"})
     foverlay = plt.figure()
-
-    atlas = mgu.get_braindata(atlas)
-    b0 = mgu.get_braindata(b0)
+    atlas = mgu.gen_utils.get_braindata(atlas)
+    b0 = mgu.gen_utils.get_braindata(b0)
     if atlas.shape != b0.shape:
         raise ValueError("Brains are not the same shape.")
```

Comment:
see my comment on jialin's PR

Reply:
see https://github.com/neurodata/ndmg/pull/372/commits/0d0f1fcea693deac7062a337bbabd64c3262d7a7 

I used "from ndmg.utils.gen_utils import get_braindata" instead of "mgu".